nixos/modules/services/security/fprintd.nix

```diff
@@ -1,14 +1,29 @@
 { config, lib, pkgs, ... }:
 let
-
   cfg = config.services.fprintd;
+
   fprintdPkg = if cfg.tod.enable then pkgs.fprintd-tod else pkgs.fprintd;
 
-in
+  # Overlay to support elanmoc2 if enabled
+  myOverlay = final: prev: {
```

Comment:
Please also override pname and version, to make the overlay more obvious.
I haven't seen this kind of overlay in NixOS modules yet, not sure whether it is okay.